src/test/store/icons.test.js

```diff
@@ -106,35 +107,46 @@ describe('actions/icons', function() {
       );
 
       validIcons.forEach((icon, i) => {
-        subject = iconsAccessors.getIconForCallNode(state, { icon });
+        subject = iconsAccessors.getIconForCallNode(
+          state,
+          _createCallNodeWithIcon(icon)
+        );
         expect(subject).toEqual(icon);
 
-        subject = iconsAccessors.getIconClassNameForCallNode(state, { icon });
+        subject = iconsAccessors.getIconClassNameForCallNode(
+          state,
+          _createCallNodeWithIcon(icon)
+        );
         expect(subject).toEqual(expectedClasses[i]);
       });
     });
   });
 
   describe('Requesting a non-existing image', function() {
     it('will not populate the local cache', async function() {
-      const actionPromise = store.dispatch(
+      const { dispatch, getState } = blankStore();
+      const actionPromise = dispatch(
         iconsActions.iconStartLoading(invalidIcon)
       );
-      expect(instances.length).toBe(1);
-      instances[0].onerror();
+      expect(imageInstances.length).toBe(1);
+      (imageInstances[0]: Object).onerror();
 
       await actionPromise;
 
-      const state = store.getState();
+      const state = getState();
       let subject = iconsAccessors.getIcons(state);
       expect([...subject]).toEqual([]);
 
-      subject = iconsAccessors.getIconForCallNode(state, { icon: invalidIcon });
+      subject = iconsAccessors.getIconForCallNode(
+        state,
+        _createCallNodeWithIcon(invalidIcon)
+      );
       expect(subject).toBeNull();
 
-      subject = iconsAccessors.getIconClassNameForCallNode(state, {
-        icon: invalidIcon,
-      });
+      subject = iconsAccessors.getIconClassNameForCallNode(
+        state,
+        _createCallNodeWithIcon(invalidIcon)
+      );
       expect(subject).toBe('');
 
       subject = iconsAccessors.getIconsWithClassNames(state);
```

Comment:
thanks for formatting this code to our newer code style :-) I can see some of my older habits from when I wrote it...